config/environments/test.rb

```diff
@@ -67,5 +67,8 @@
   config.x.hses.auth_base = "https://staging.hses.ohs.acf.hhs.gov"
   config.x.hses.client_id = Rails.application.credentials.hses_client_id
   config.x.hses.client_secret = Rails.application.credentials.hses_client_secret
-  config.x.bypass_auth = ENV["BYPASS_AUTH"] == "true"
+  config.x.bypass_auth = ENV.fetch("CT_BYPASS_AUTH", "false") == "true"
```

Comment:
[suggestion] as above, `ENV["CT_BYPASS_AUTH"] == "true"` should work as expected